src/components/TopNavigationBar.js

```diff
@@ -1,29 +1,35 @@
-import logo from "../assets/sparrow-logo.svg"
+import logo from "../assets/sparrow-logo-w-bg.png"
 import { signOutUser, removeUser } from "../config/googleSignIn"
 import { Link } from "react-router-dom"
+import Container from "react-bootstrap/Container"
+import Navbar from "react-bootstrap/Navbar"
+import Button from "react-bootstrap/Button"
+import Image from "react-bootstrap/Image"
+import style from "./TopNavigationBar.module.css"
 
 function TopNavigationBar({ currentUser }) {
+  const signedInUser = currentUser ? currentUser.displayName : "user"
+
   // When signing out removes user from 'users' collection and signs out user.
   const handleSignOutUser = () => {
     removeUser(currentUser)
     signOutUser()
   }
   return (
     <>
-      {/* Sparrow logo is HUGE, put some temporary inline styling */}
-      <img src={logo} alt='Sparrow Logo' style={{ height: "28px" }} />
-
-      {/* Search input and button */}
-      {/* <input type="text" placeholder='Search chat for...' /> */}
-      {/* <button>Search</button> */}
-
-      {/* Profile */}
-      <span> Placeholder Username </span>
-
-      {/* Logout button */}
-      <Link to='/'>
-        <button onClick={handleSignOutUser}>Logout</button>
-      </Link>
+      <Navbar className={style.navbarContainer}>
+        <Container>
+          <Image src={logo} width='75' />
+          <Image src={currentUser.photoURL} width='75' />
```

Comment:
```suggestion
          <Image src={currentUser ? currentUser.photoURL : "some_placeholder_img"} width='75' />
```
Right now we have a problem of the application crashing when the user signs in, because our router redirects the user to chat as soon the sign in button is clicked. Once we implement #59 we won't have this error anymore.